src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -219,7 +221,29 @@ private static boolean isToolTip(Node currentNode) {
         
         return false;
     }
-
+    
+    private static boolean isMotdComponent(Node currentNode) {      
+        if (currentNode.getNodeName().equalsIgnoreCase("script")) {
+            return currentNode.getTextContent().contains("closeMotd");
+            
+        } else if (currentNode.getNodeName().equalsIgnoreCase("div")) {
+            NamedNodeMap attributes = currentNode.getAttributes();
+            
+            if (attributes == null) { 
+                return false;
+            }
+                
+            for (int i = 0; i < attributes.getLength(); i++) {
```

Comment:
Right, I digged further and found some resources on this:

http://stackoverflow.com/questions/4171380/generic-foreach-iteration-of-namednodemap

Apparently the name contains "map" but does not implements java.lang.iterable so we can't do that.